database/sql/V072__auction_solution_orders.sql

```diff
@@ -0,0 +1,36 @@
+-- Table to store all proposed solutions for an auction, received from solvers during competition time.
+-- A single auction can have multiple solutions, and each solution can contain multiple orders.
+-- Order is not allowed to be part of more than one solution within the same auction.
+
+-- `solver` + `solution_id` are unique within an auction.
+
+-- `deadline` is used to determine if the order is part of a winning solution.
+
+CREATE TABLE auction_solution_orders (
+   auction_id bigint NOT NULL,
+   solver bytea NOT NULL,
+   solution_id string NOT NULL,
+   order_uid bytea NOT NULL,
+   -- The block number until which the order should be settled.
+   -- Not NULL for winning orders.
+   deadline bigint,
```

Comment:
Can we comply with basic schema normalisation (ie. if solver is always the same for solution id, have it in a separate table).

Reply:
Initially did it that way but didn't like it because of a lot of duplicated data used for primary keys. Will adjust since from Martin's comments I see he would prefer it as well.